Python/optimizer_analysis.c

```diff
@@ -301,6 +301,14 @@ sym_set_null(_Py_UOpsSymType *sym)
     sym_set_flag(sym, KNOWN);
 }
 
+static inline void
+sym_set_const(_Py_UOpsSymType *sym, PyObject *const_val)
+{
+    sym_set_type(sym, Py_TYPE(const_val));
+    sym_set_flag(sym, TRUE_CONST | KNOWN | NOT_NULL);
+    Py_XSETREF(sym->const_val, Py_NewRef(const_val));
```

Comment:
This is temporary, just ignore this for now. I will remove it in https://github.com/python/cpython/pull/115817.